libs/design-system/web-component-card/src/lib/web-component-card.component.scss

```diff
@@ -0,0 +1,58 @@
+:host {
+  display: block;
+  position: relative;
+  border-radius: 0.5rem;
+  overflow: hidden;
+  background-color: var(--sys-surface-container-low);
+  box-shadow: 0rem 0.3125rem 1rem 0rem rgb(from var(--sys-shadow) r g b / 0.24);
+
+  img {
+    height: 15rem;
+    width: 100%;
+  }
+
+  .card-content {
+    padding: 1.5rem 1.5rem 0 1.5rem;
+  }
+
+  .default_style {
```

Comment:
Replace `.default_style`, `.style_change`, and `.web-component-name` with the following:
```scss
.product-title,
.web-component-name {
  font: var(--sys-title-large);
  letter-spacing: var(--sys-title-large-tracking);
  color: var(--sys-on-background);
  margin: 0;
}

.product-title:has(+ .web-component-name) {
  font: var(--sys-title-medium);
  letter-spacing: var(--sys-title-medium-tracking);
  color: var(--sys-inverse-surface)
  margin-bottom: 0.5rem;
}
```

This uses the [`:has()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:has) selector to detect when both a product title and component name is present.
Also `line-height` is automatically set by `font: ...` so you don't need to set it yourself